src/frontend/shortlist/src/components/layout/signup_login/Signupview.vue

```diff
@@ -0,0 +1,127 @@
+<script>
```

Comment:
I would suggest keeping the file names without "view" and "View" so that later we don't have to work on renaming and organizing things. For example, keep Login.vue and avoid LoginView.vue or Loginview.vue.

Reply:
This is a good comment. Yes, I will change the name.